### PR TITLE
document newline normalization in literal strings

### PR DESCRIPTION
**Pull request by stevengj**
This documents #14073.

### CODE REVIEW

**Comment by tkelman** [comment]
Great, thanks.

**Comment by stevengj** [comment]
The AppVeyor failure (in the `parallel` tests) is surely unrelated.    Is there an issue for this?   These intermittent failures are annoying (or worse).

**Comment by tkelman** [comment]
That looks like a possible culprit. Have there been gc and/or inference changes recently? That assertion failure along with a smallvector llvm assertion failure have gotten frequent within the last week or so on 32 bit, and timeouts where one of the workers freezes partway through have been happening often on 64 bit.

File: doc/manual/strings.rst [hunk 1]
```diff
@@ -439,7 +439,7 @@ backslash:
     julia> print("I have \$100 in my account.\n")
     I have $100 in my account.
 
-Triple-Quoted Strings Literals
+Triple-Quoted String Literals
 ------------------------------
```
Comment (tkelman) [inline doc/manual/strings.rst]: RST is picky about header lengths

Comment (stevengj) [inline doc/manual/strings.rst]: Pushed a fix.

**Pull request merged**